feature-policy/header-policy-payment-disabled.sub.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+  if (window.testRunner) {
+    testRunner.dumpAsText();
+    testRunner.waitUntilDone();
+  }
+</script>
+<iframe id="same-origin" src="resources/vibrate-disabled.html"></iframe>
+<iframe id="cross-origin" src="{{location[scheme]}}://{{domains[www]}}:{{location[port]}}/feature-policy/resources/vibrate-disabled.html"></iframe>
+<iframe id="same-origin" src="resources/vibrate-disabled.html"></iframe>
```

Comment:
(Or else they should be deleted -- they look identical to the other two lines, so I'm not sure if they're there for a reason, or if it is accidental.)